composer.json

```diff
@@ -11,17 +11,19 @@
             "email": "dcramer@gmail.com"
         }
     ],
-    "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.1",
-        "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
-    },
     "require": {
         "php": ">=5.5",
         "ext-hash": "*",
         "ext-json": "*",
         "ext-mbstring": "*",
-        "ext-curl": "*"
+        "ext-curl": "*",
+        "composer/ca-bundle": "~1.0",
+        "symfony/options-resolver": "~2.8|~3.0"
+    },
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "~2.1",
+        "phpunit/phpunit": "^4.8 || ^5.0",
+        "monolog/monolog": "*"
```

Comment:
Indeed, I just moved the whole block of code after the `require` section. The scope of this PR is not to change these dependencies